.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+## Version
+[provide output of the `$ openshift version` command]
+[provide output of the `$ oc version` command and the version of the master]
```

Comment:
how about "[provide output of the `$ openshift version` or `$ oc version` command]"

i really don't like to complicate this with requesting the master version because it is not easy to get that information.